src/Illuminate/View/Engines/CompilerEngine.php

```diff
@@ -56,8 +56,18 @@ public function get($path, array $data = [])
         // Once we have the path to the compiled file, we will evaluate the paths with
         // typical PHP just like any other templates. We also keep a stack of views
         // which have been rendered for right exception messages to be generated.
-        $results = $this->evaluatePath($compiled, $data);
-
+        $results  = $this->evaluatePath($compiled, $data);
+        if (env("THEME_HINT_ENABLED")) {
```

Comment:
> But I assume .env file seems to be simpler than config\view.php.

`.env` is not loaded when you're caching the config, which is the normal practise for production. So what would be the use of this when it not going to be parse for production.

And what up with the code example inside the view. We don't dictate that user has to use Twitter Bootstrap.